### DISCUSSION TITLE
MultiDC cluster, observing 100% CPU when connecting kafka debezium source conector

### DISCUSSION THREAD

**Discussion by amitesh88** [scylladb/scylla-machine-image#506] [Q&A]
We have our multi dc setup with 3 node in dc1 and 3 in dc2 . Although both the DCs are in the same region n same subnet, This is done as we require separate clusters for reading n writing data.

Our setup creates a new table everyday at 12 midnight with cdc enabled in DC1

At the same time we also create a kafka debezium source connector to consume cdc logs from DC2 everyday

Issue:
At around 12
When creating a new source con , we observe scylladb servers on dc2 consumes 100% cpu.

We increased cpu from 16cores to 32 but still same behavior
Once the kafka connector creates its topic and start reading data from cdc log ,scylladb cpu cools down

Logs:

The logs in syslog shows reader_concurrency_semaphores for that time period.

Any expert thoughts is appreciated
Thanks in advance

**Comment by fruch** [comment]
This doesn't sound related to this repository

I would recommend raising an issue on the CDC connector repository

And attach logs of the connector and scylla

**Comment by mykaul** [comment]
@amitesh88 - isn't this a duplicate of https://github.com/scylladb/scylla-cdc-source-connector/issues/47 ?

**Reply by amitesh88** [reply]
Yes its the same issue .Initially posted here but redirected to https://github.com/scylladb/scylla-cdc-source-connector/issues/47
